source/crud/delete.txt

```diff
@@ -51,6 +51,14 @@ To delete all documents from a collection, pass an empty
    more information and examples, see
    :method:`~db.collection.deleteMany()`.
 
+.. note::
+
+   If you want to delete all documents from a large collection, dropping
```

Comment:
Since you want to direct attention to dropping the collection, perhaps it makes sense to link to that too?

```suggestion
   If you want to delete all documents from a large collection, dropping with the :method:`db.collection.drop` method.
```

Reply:
Done! Thanks for this suggestion!